### ISSUE TITLE
Some noise is heard on the local side

### ISSUE THREAD

**Issue by izilzty** [rustdesk/rustdesk#8375]
### Bug Description

Hi, if the default format of the local side sound card is greater than 48000 Hz, the local side will hear some noise when the remote side plays audio.

### How to Reproduce

Set the local side sound card default format to be greater than 48000 Hz, and keep the remote side sound card to default (less than or equal to 48000 Hz).
![local side format](https://github.com/rustdesk/rustdesk/assets/18079946/695922a0-c76b-47ac-93a6-6754f8ffe285)

### Expected Behavior

Clear sound without noise

### Operating system(s) on local side and remote side

Windows 11 23H2 -> Windows 10 LTSC 2021

### RustDesk Version(s) on local side and remote side

1.2.6 -> 1.2.6

### Screenshots

[local side noise.zip](https://github.com/user-attachments/files/15858786/local.side.noise.zip)

**Comment by rustdesk** [comment]
@fufesou

**Comment by rustdesk** [comment]
Could you help test 1.2.5? I wanna confirm if it is related to a latest code change.

**Comment by fufesou** [comment]
@izilzty Hi, do you use voice calls? Oddly enough, the speaker properties affect how playing the sound.

**Comment by izilzty** [comment]
I tested 1.2.5->1.2.5 and 1.2.3-2->1.2.3-2, both have the same problem on my computer, and I haven't tried voice calls yet.

**Comment by izilzty** [comment]
I'm guessing this might be an issue on the local side of the Windows version, because if I keep the current configuration and use the iOS app to connect to both computers to play audio, they both sound clear.

**Comment by FiveKnive** [comment]
I have the same issue!

**Comment by rustdesk** [comment]
How and when you can hear the noise?

**Comment by FiveKnive** [comment]
Always when I play some sort of audio it sounds like it is distorted with this noise. You can hear it a little bit in the linked zip file of this issue. Happens when I connect from my Windows PC to another Windows PC or to Linux.

**Comment by rustdesk** [comment]
> Set the local side sound card default format to be greater than 48000 Hz, and keep the remote side sound card to default (less than or equal to 48000 Hz).

How about your local side default settings?

**Comment by FiveKnive** [comment]
![image](https://github.com/user-attachments/assets/f2a65aa2-a3ca-4b92-abff-664361d4540a)

Ok if I set it to 48000Hz on my client pc it works. But now I have problems with my other audio programs and can not have the full potential of my DAC :(
Is there no easy fix for this. It worked with all other remote softwares I tried.

**Comment by rustdesk** [comment]
What's you old settings?

**Comment by FiveKnive** [comment]
My setting is 192000 Hz. The remote side is 48.000 Hz.

**Comment by 21pages** [comment]
The strange thing is that I don't hear any noticeable noise from the attached file, Is it slight  noise?

**Comment by FiveKnive** [comment]
Yes it is a bit short and difficult to hear in the example above. I recorded a longer file. You will hear it in the background if you have a decent audio setup.
[noise-rustdesk.zip](https://github.com/user-attachments/files/16537390/noise-rustdesk.zip)

**Comment by izilzty** [comment]
Hi, the attached audio is from Windows audio settings, you can play it on your computer and compare it with the attached audio file. Its location can be found from the link below.
> https://webcammictest.com/headphones/guides/test-in-windows/

**Comment by FiveKnive** [comment]
> Hi, the attached audio is from Windows audio settings, you can play it on your computer and compare it with the attached audio file. Its location can be found from the link below.
> 
> > https://webcammictest.com/headphones/guides/test-in-windows/

It sounds broken over Rustdesk for me too. It sounds clean over rdp.

**Comment by FiveKnive** [comment]
> The strange thing is that I don't hear noticeable noise from the attached file, is it slight noise?

Can you hear it now? I tested it with my friends and they have the same issue on their machines.

**Comment by 21pages** [comment]
> The remote side is 48.000 Hz.

Could you record the same song with local settings 48000Hz, I want to compare this two file.

**Comment by FiveKnive** [comment]
Sure
[rustdesk-clean-48k.zip](https://github.com/user-attachments/files/16568364/rustdesk-clean-48k.zip)

**Comment by jonas-herkelmedia** [comment]
Hi, will this be fixed in the next version? I could help if you need some more details on the sound settings where the problem occurs.

**Comment by 21pages** [comment]
Sorry for the late, will continue this issue after completing the work at hand.
